docs/dev/arch_decision.md

```diff
@@ -0,0 +1,7 @@
+# Architecture Decision Records
+
+The intention is to document deviations from a standard Model View Controller (MVC) design.
+
+!!! warning "Developer Note - Remove Me!"
+    Optional page, remove if not applicable.
+    For examples see [Golden Config](https://github.com/nautobot/nautobot-plugin-golden-config/tree/develop/docs/dev/dev_adr.md) and [nautobot-plugin-reservation](https://github.com/networktocode/nautobot-plugin-reservation/blob/develop/docs/dev/dev_adr.md).
```

Comment:
Should we remove this?

Reply:
I updated the documentation where I was possible to to do. If you can replace this warning with some documentation, would be great. Otherwise, we can remove that.

Reply:
Moved to:
https://github.com/nautobot/nautobot-plugin-ssot/pull/255